stdlib/public/TensorFlow/ArrayOps.swift

```diff
@@ -0,0 +1,114 @@
+//===-- ArrayOps.swift ----------------------------------------*- swift -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This file contains some Array ops that cannot be properly handled by #tfop.
+//
+// TODO: These should be deleted once we can properly generate raw ops for these.
+//
+//===----------------------------------------------------------------------===//
+
+import CTensorFlow
+
+public extension Raw {
+
+/// Saves tensors in V2 checkpoint format.
+///
+/// By default, saves the named tensors in full.  If the caller wishes to save
+/// specific slices of full tensors, "shape_and_slices" should be non-empty strings
+/// and correspondingly well-formed.
+///
+/// - Parameters:
+///   - prefix: Must have a single element. The prefix of the V2 checkpoint to which we
+///     write the tensors.
+///   - tensor_names: shape {N}. The names of the tensors to be saved.
+///   - shape_and_slices: shape {N}.  The slice specs of the tensors to be saved.
+///     Empty strings indicate that they are non-partitioned tensors.
+///   - tensors: `N` tensors to save.
+@inlinable @inline(__always)
+static func saveV2(
```

Comment:
Fix indentation

Reply:
This code conforms to the style of the generated Raw ops.

Reply:
What do you mean by “conforms to that”? This is really backwards. There’s no “style” or “convention” of raw ops. It’s just the reality of op generation. But since this what you wrote by hand, this code needs to follow the Swift code style.